packages/client/hmi-client/src/components/widgets/tera-related-publications.vue

```diff
@@ -61,10 +61,12 @@ import Dialog from 'primevue/dialog';
 import DataTable from 'primevue/datatable';
 import Column from 'primevue/column';
 import { ResourceType } from '@/types/common';
-import { profileDataset, profileModel, fetchExtraction } from '@/services/knowledge';
+import { profileDataset, profileModel, fetchExtraction, alignModel } from '@/services/knowledge';
+import { PollerResult } from '@/api/api';
+import { isEmpty } from 'lodash';
 
 const props = defineProps<{
-	publications?: Array<{ name: string; id: string | undefined }>;
+	publications?: Array<{ name: string | undefined; id: string | undefined }>;
 	relatedPublications?: Array<{ name: string; id: string | undefined }>;
```

Comment:
Are those `Documents` or `Publications`. If it's the former, we need to rename those props.

Reply:
I guess we were using Artifacts before, but I agree the naming is confusing regardless.  Going to rename not only the props but the entire modal itself.